<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSettingsQt</class>
 <widget class="QDialog" name="SoundSettingsQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtres.qrc">
    <normaloff>:/icon/img/rpmb_icon_64.png</normaloff>:/icon/img/rpmb_icon_64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sound File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="filenameEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="filenameBrowseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Volume modifier:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="soundVolumeSlider">
          <property name="minimum">
           <number>-10</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="soundVolumeDbLabel">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+0 dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCrop">
     <property name="title">
      <string>Crop Sound</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start sound at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startSoundValueSpin">
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="startSoundUnitCombo">
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Stop sound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopSoundAtAfterCombo"/>
        </item>
        <item>
         <widget class="QSpinBox" name="stopSoundValueSpin">
          <property name="maximum">
           <number>10000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopSoundUnitCombo"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previewSoundButton">
     <property name="text">
      <string>Preview Sound</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SoundSettingsQt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SoundSettingsQt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
